src/plugins/guided_onboarding/public/components/guide_panel.tsx

```diff
@@ -57,7 +57,7 @@ export const GuidePanel = ({ api, application, notifications, theme$ }: GuidePan
   const [pluginState, setPluginState] = useState<PluginState | undefined>(undefined);
   const [guideConfig, setGuideConfig] = useState<GuideConfig | undefined>(undefined);
   const [isLoading, setIsLoading] = useState<boolean>(false);
-  const { darkMode: isDarkTheme } = useObservable(theme$, { darkMode: false });
+  const { darkMode: isDarkTheme } = useObservable(theme$, { darkMode: false, name: 'amsterdam' });
```

Comment:
I was really hoping to avoid this kind of defaulting.  It also requires review from too many teams when we change it.

I realize the `useObservable` hook requires a default, but getting the dark mode value shouldn't require passing in the name of the theme, not to mention this name may be inaccurate.

Either we need to introduce a new hook to access the `theme$` observable, or we need to make it optional.  Perhaps the former.  What do you think @tsullivan ?